contributing_to_docs/doc_guidelines.adoc

```diff
@@ -746,18 +748,36 @@ There might be advanced networking examples that require specific IP addresses,
 == Links, hyperlinks, and cross references
 Links can be used to cross-reference internal assemblies or send readers to external information resources for further reading.
 
-In OpenShift docs:
+In OpenShift Container Platform docs:
 
-* All links to internal content is created using `xref` and **must have an anchor ID**.
+* All links to internal content is created using `xref` and each `xref` **must have an anchor ID**.
 * Only use `xref` in assemblies, not in modules.
-* All links to external websites are created using `link`.
-* Links between different distros, such as from ROSA to OpenShift Container Platform, are external links and not cross-references. These external links use `link` instead of `xref` and need to be in distro-specific files or conditionalized to apply to the relevant distros.
+* All links to external websites are created by using `link`.
+* Links between different distributions, or _distros_, such as from ROSA to OpenShift Container Platform, are external links and not cross-references. These external links use `link` instead of `xref` and need to be in distro-specific files or a conditionalized state to apply to the relevant distros.
 
 [IMPORTANT]
 ====
-Do not split link paths across lines when wrapping text. This will cause issues with the doc builds.
+Do not split link paths across lines when wrapping text. This will cause issues with the documentation builds.
 ====
 
+[id="lead-in-link-sentences"]
+=== Lead-in link sentences
+
+To maintain consistency across OpenShift Container Platform documentation, ensure that you adhere to the following guidance for lead-in link sentences:
+
+* Maintain the "For more information, see <section_name>" format when pointing to a section in the same document.
+* Wrap the _<section_name>_ in quotation marks, where _<section_name>_ can refer to either a module or an assembly.
+* Do not state "in the Additional Resources section" or other variants. Mentioning the _<section_name>_ is enough to prompt a documentation user to find the section in the same document.
+* When pointing to a link that exists in another customer document, use the "See "<section_name>" in _<document_name>_" format. Note the quotation marks around _<section_name>_ and the italics around _<document_name>_.
+* You can add link sentences to either the _Additional resources_ section or the _Next steps_ section, but know the following differences:
+** Additional resources - module level: This section is optional. The listed resources link to other material closely related to the contents of the module. Focus on relevant resources that might interest the user. Do not list resources for completeness
+** Additional resources - assembly level: Similar to the previous item, but if a resource applies to all of the modules in a user story, consider listing the resource in the _Additional resources_ section of the assembly file. If you do this, consider mentioning the resource in the module as a comment.
```

Comment:
For these two, consider showing their different markup (`.Additional resources` vs. `== Additional resources`) to help identify/clarify